odoo/tools/config.py

```diff
@@ -335,6 +332,19 @@ def _build_cli(self):
                          help="Absolute path to the GeoIP Country database file.")
         parser.add_option_group(group)
 
+        group = optparse.OptionGroup(parser, "Shell options")
+        group.add_option(
+            '--shell-file', dest='shell_file',
```

Comment:
In another work I'm changing the default value of all path commands to an empty string
```suggestion
            '--shell-file', dest='shell_file', my_default='',
```